cnames_active.js

```diff
@@ -911,6 +911,7 @@ var cnames_active = {
   "eco": "quick-eco.netlify.app",
   "ecstar": "ecstar-js.github.io/Ecstar",
   "ed2k": "sunnyli.github.io/ed2k.js",
+  "edgeware-blog": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "edgeware-blog": "cname.vercel-dns.com", // noCF
```
As the CI error notes, you'll want the Cloudflare proxy disabled when using Vercel